torch/utils/tensorboard/__init__.py

```diff
@@ -0,0 +1,6 @@
+try:
```

Comment:
If this is the module that most end users are going to use, it absolutely needs a docblock. Take a look at, e.g., https://pytorch.org/docs/stable/hub.html for what level of documentation we're going for here.

Reply:
Added more thorough docs to `writer.py`, since that's what is exposed to the user. Is that sufficient?